docs/ide/tutorial-csharp-aspnet-core.md

```diff
@@ -126,11 +126,11 @@ This solution follows the **Razor Page** design pattern. It is different than th
 
 1. Keep the browser window open and return to Visual Studio.
 
-1. In Visual Studio, choose **About.cshtml**. Then, delete the word _changed_ and in its place, add the words _file and directory_.
+1. In Visual Studio, choose **About.cshtml**. Then, delete the word _additional_ and in its place, add the words _file and directory_.
 
```

Comment:
@jmayntzhusen This is a good edit for technical accuracy; I'd like to keep this edit. :)